docs/contributing-to-airbyte/working-with-airbyte-platform.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Working with Airbyte Platform
+---
+
+# Overview
+
+In order to facilitate faster development workflows, airbytehq/airbyte was split into two repositories on 2023-19-02:
+
+- airbytehq/airbyte connectors and development around
+- airbytehq/airbyte-platform the codebase that facilitates connectors
+- airbytehq/airbyte-protocol used by both connectors and platform, the underlying protocol that airbyte uses to perform data transfers
+
+If you have an existing pull request in airbytehq/airbyte that should instead target airbyte-platform there are two ways that you can move it over
+
+# Issues and Pull Requests
+
+Please open issues in airbytehq/airbyte and tag them platform. Issues have been disabled on airbyte-platform in order to keep issue tracking in one centralized location
+Please open pull requests targeting airbytehq/airbyte-platform in airbytehq/airbyte-platform and reference the issue in airbytehq/airbyte
+
+# Migrating from airbytehq/airbyte
```

Comment:
We had discussed not including this workflow since this is a very temporary requirement. Curious as to why we decided to add it now (did we get support tickets?)

Reply:
I can leave this out for now and put it back if we get requests about it!